js/components/adPanel.js

```diff
@@ -90,13 +103,30 @@ var AdPanel = React.createClass({
     }
 
     var isLive = this.props.currentAdsInfo.currentAdItem.isLive;
+    var isSSAI = this.props.currentAdsInfo.currentAdItem.ssai;
 
     if (this.props.skinConfig.adScreen.showAdCountDown) {
-      var remainingTime;
+      var remainingTime = 0;
       if (isLive) {
-        remainingTime = parseInt(
-          (this.props.adStartTime + this.props.adVideoDuration * 1000 - new Date().getTime()) / 1000
-        );
+        remainingTime = parseInt((this.props.adStartTime + this.props.adVideoDuration * 1000 - new Date().getTime())/1000);
+        if (isSSAI) {
```

Comment:
@ipadilla4 is this behavior actually correct? It seems like you are trying to use this state to keep track of the amount of time the ad was paused and add it to the timer, is that correct? If so the result would be the timer is always increasing when the ad is paused... is that the expected user experience? Is that what non-ssai ads do currently?

Reply:
@dallen42 for live non-ssai ads we use current time (new Date().getTime()) to calculate remaining time. Since now we can pause our live ads, we need a way to calculate the total time it was paused so we can add it to remaining time, so when user resume the counter keeps decreasing from the time was paused.

Reply:
@dallen42 or maybe we can preserve the time when ad was paused and then decrease that number every second when user resumes the ad. I can try this approach with props and see how it behaves.